iOS-Email-Client/Controllers/ComposeViewController.swift

```diff
@@ -1047,6 +1051,7 @@ extension ComposeViewController: CNContactPickerDelegate {
             
             view.remove(token)
             view.add(token, highlight: textColor, background: bgColor)
+            self.checkEnableSendButton()
```

Comment:
You were removing the token, that triggers a check for the send button, but adding one does not so the button remained disabled